src/measure_theory/lp_space.lean

```diff
@@ -321,6 +322,26 @@ begin
   { exact snorm'_mono_ae ennreal.to_real_nonneg h }
 end
 
+lemma snorm_le_of_bound {f : α → E} {C : ℝ} (hC : 0 ≤ C) (hfC : ∀ᵐ x ∂μ, ∥f x∥ ≤ C) :
+  snorm f p μ ≤ ((μ set.univ) ^ p.to_real⁻¹) * (@coe ℝ≥0 ℝ≥0∞ _ ⟨C, hC⟩) :=
```

Comment:
why did you prefer `(@coe ℝ≥0 ℝ≥0∞ _ ⟨C, hC⟩)` over `ennreal.of_real C` ?
Also it looks like `hC` is not necessary since it can be deduced from `hfC`.

Reply:
The same proof, but with `ennreal.of_real C` in the goal and no `hC` hypothesis:
```
lemma snorm_le_of_bound {f : α → F} {C : ℝ} (hfC : ∀ᵐ x ∂μ, ∥f x∥ ≤ C) :
  snorm f p μ ≤ ((μ set.univ) ^ p.to_real⁻¹) * (ennreal.of_real C) :=
begin
  by_cases hμ : μ = 0,
  { simp [hμ] },
  haveI : μ.ae.ne_bot := ae_ne_bot.mpr hμ,
  by_cases hp : p = 0,
  { simp [hp] },
  have hC : 0 ≤ C, from le_trans (norm_nonneg _) hfC.exists.some_spec,
  have hC' : ∥C∥ = C := by rw [real.norm_eq_abs, abs_eq_self.mpr hC],
  have : ∀ᵐ x ∂μ, ∥f x∥ ≤ ∥(λ _, C) x∥, from hfC.mono (λ x hx, hx.trans (le_of_eq hC'.symm)),
  convert snorm_mono_ae this,
  rw [snorm_const _ hp hμ, mul_comm, ← of_real_norm_eq_coe_nnnorm, hC', one_div],
end
```
I also replaced `E` by `F`, which is also a normed group etc, but is not a measurable space.

Reply:
I believe that if `α` is empty, `hC` can't be deduced from `hfC`.  But using `ennreal.of_real C` gets around that need nicely, thanks!